src/reactIntl.js

```diff
@@ -0,0 +1,10 @@
+export const intl = {
+	formatMessage: jest.fn(),
```

Comment:
Nice! This has also been on my mind to do.

What do you think about supplying a mock function to `jest.fn()` so that these method return values? i.e.,

`formatMessage: jest.fn(() => 'mock format message')` 

The main use case for this would be snapshots, so that there is some content where copy would typically be.

Reply:
Agreed, i got thrown off when i saw 'undefined' in a snapshot yesterday because formatMessage return `undefined` without that. I'll update later today